_security/authentication-backends/basic-authc.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: HTTP basic authentication
+parent: Authentication backends
+nav_order: 46
+redirect_from:
+---
+
+
+# HTTP basic authentication
+
+When Security is configured for HTTP basic authentication, it provides a simple challenge and response process for gaining access to OpenSearch and its resources, which prompts you to sign in with a username and password. To set up HTTP basic authentication, you must enable it in the `http_authenticator` section of the configuration by specifying `type` as `basic`, as shown in the following example:
+
+```yml
+authc:
+  basic_internal_auth_domain:
+    description: "Authenticate via HTTP basic against internal users database"
+    http_enabled: true
+    transport_enabled: true
```

Comment:
This setting may no longer be needed and I filed an issue in the security repo to determine if it can be removed safely: https://github.com/opensearch-project/security/issues/3191